config/initializers/new_framework_defaults.rb

```diff
@@ -6,20 +6,20 @@
 #
 # Read the Guide for Upgrading Ruby on Rails for more info on each option.
 
-Rails.application.config.action_controller.raise_on_unfiltered_parameters = true
+Rails.application.config.action_controller.raise_on_unfiltered_parameters = false
```

Comment:
For this file, should we be deleting it after we know that flipping the switches doesn't break anything?

Reply:
We can't delete this yet. I think Rails 5.0 generates this file for new apps too.

https://github.com/rails/rails/pull/28469

We can remove this in 5.1 by calling `config.load_defaults` in `application.rb`